src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDynamicMemberReferenceExpression.cs

```diff
@@ -0,0 +1,374 @@
+                // CS1001: Identifier expected
+                //             int i = /*<bind>*/d./*</bind>*/;
+                Diagnostic(ErrorCode.ERR_IdentifierExpected, ";").WithLocation(11, 44)
+            };
+
+            VerifyOperationTreeAndDiagnosticsForTest<MemberAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void IDynamicMemberReferenceExpression_SimpleMethodCall()
+        {
+            string source = @"
+using System;
+
+namespace ConsoleApp1
+{
+    class C1
+    {
+        static void M1()
+        {
+            dynamic d = null;
+            /*<bind>*/d.GetValue()/*</bind>*/;
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IOperation:  (OperationKind.None) (Syntax: 'd.GetValue()')
+  Children(1):
+      IDynamicMemberReferenceExpression (Member name: GetValue, Containing Type: null) (OperationKind.DynamicAccessExpression, Type: dynamic) (Syntax: 'd.GetValue')
+        Instance Receiver: ILocalReferenceExpression: d (OperationKind.LocalReferenceExpression, Type: dynamic) (Syntax: 'd')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<InvocationExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void IDynamicMemberReferenceExpression_InvalidMethodCall_MissingName()
+        {
+            string source = @"
+namespace ConsoleApp1
+{
+    class C1
+    {
+        static void M1()
+        {
+            dynamic d = null;
+            /*<bind>*/d.()/*</bind>*/;
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'd.()')
+  Children(1):
+      IDynamicMemberReferenceExpression (Member name: , Containing Type: null) (OperationKind.DynamicAccessExpression, Type: dynamic, IsInvalid) (Syntax: 'd.')
+        Instance Receiver: ILocalReferenceExpression: d (OperationKind.LocalReferenceExpression, Type: dynamic) (Syntax: 'd')
+";
+            var expectedDiagnostics = new DiagnosticDescription[] {
+                // CS1001: Identifier expected
+                //             /*<bind>*/d.()/*</bind>*/;
+                Diagnostic(ErrorCode.ERR_IdentifierExpected, "(").WithLocation(9, 25)
+            };
+
+            VerifyOperationTreeAndDiagnosticsForTest<InvocationExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void IDynamicMemberReferenceExpression_InvalidMethodCall_MissingCloseParen()
+        {
+            string source = @"
+namespace ConsoleApp1
+{
+    class C1
+    {
+        static void M1()
+        {
+            dynamic d = null;
+            /*<bind>*/d.GetValue(/*</bind>*/;
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'd.GetValue(/*</bind>*/')
```

Comment:
Apparently I misunderstood what that PR was implementing, it's actually a BoundDynamicInvocation, tracked by https://github.com/dotnet/roslyn/issues/20114.